tests/integration/linux_bridge_test.py

```diff
@@ -269,6 +303,15 @@ def test_replace_port_on_linux_bridge(port0_vlan101, port1_up):
         )
 
 
+def test_linux_bridge_over_bond_over_slave(bond0, bridge0):
+    bridge_state = bridge0[Interface.KEY][0]
+    desired_state = deepcopy(bond0)
```

Comment:
I think this is the responsibility of the fixture.

Reply:
The linux bridge is no longer a fixture, and instead it's state is generated within the test itself; as such, I think this no longer applies.